.github/workflows/workspace.yaml

```
@@ -22,15 +22,16 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: fmt
-          args: --all -- --check
+          args: --manifest-path trusted/Cargo.toml --all -- --check
```

Why not use the working dir override?
```suggestion
          args: --all -- --check
        working-directory: ./trusted
```

---

I wanted to :(, https://github.com/actions-rs/cargo/issues/6

---

Fair enough. Seems worth documenting here

---

added a comment